laravelServer/server/resources/views/welcome.blade.php

```diff
@@ -63,10 +63,10 @@
                                         {{ csrf_field() }}
                                         <input type="checkbox" name="status" class="onoffswitch-checkbox" id="placeholder2" tabindex="0" onchange="this.form.submit()" {{ $statuses->where('type', 2)->where('on', true)->isNotEmpty() ? 'checked' : '' }}>
                                         <label class="onoffswitch-label" for="placeholder2"></label>
-                                        <input type="hidden" name="type" value="2">
+                                        <input type="hidden" name="type" value="stoel">
```

Comment:
currently type is a smallInt type of pgsql. So if we try to upload this our database will say wtf do you mean VARCHAR. wtf even is that. I want an int between -32,768 and 32,767